cedar-policy-core/src/parser/err.rs

```diff
@@ -48,6 +48,71 @@ pub enum ParseError {
     #[error("poorly formed: {0}")]
     #[diagnostic(code(cedar_policy_core::parser::to_ast_err))]
     ToAST(String),
+
+    /// Error in the CST -> AST transform, internal invariant violation.
+    #[error("poorly formed: {0}")]
+    #[diagnostic(code(cedar_policy_core::parser::to_ast_err))]
+    InternalInvariantViolation(String),
+
+    /// Error in the CST -> AST transform, too many `-`s.
+    #[error("too many -'s: {0}")]
+    #[diagnostic(code(cedar_policy_core::parser::to_ast_err))]
+    TooManyDashes(String),
+
+    /// Error in the CST -> AST transform, too many `!`s.
+    #[error("too many !'s: {0}")]
+    #[diagnostic(code(cedar_policy_core::parser::to_ast_err))]
+    TooManyBangs(String),
```

Comment:
Remove the string argument from these. The current error message will be `"too many -'s: too many -'s"`

Reply:
Apart from Kesha's comment, please see my comment above.